### ISSUE TITLE
Error: Cannot find module 'react-render-hook'

### ISSUE THREAD

**Issue by faceyspacey** [bruderstein/unexpected-react#4]
I'm getting that error while using the example app. `react-render-hook` appears to be there along with the react dev tools. I'm not sure what the problem is.

**Comment by faceyspacey** [comment]
I added `react-render-hook` manually to the root package.json file of the example project, and now the `SelectOptions` tests pass, but all 6 `Select` tests all fail with this error:

```
1) Select should render a div.Select:
     TypeError: Cannot read property 'data' of undefined
      at node_modules/unexpected-react/node_modules/unexpected-htmllike-reactrendered-adapter/lib/RenderedReactElementAdapter.js:77:42
      at Array.map (native)
      at RenderedReactElementAdapter.getChildren (node_modules/unexpected-react/node_modules/unexpected-htmllike-reactrendered-adapter/lib/RenderedReactElementAdapter.js:75:47)
      at diffElement (node_modules/unexpected-react/node_modules/unexpected-htmllike/lib/diff.js:238:90)
      at diffElementOrWrapper (node_modules/unexpected-react/node_modules/unexpected-htmllike/lib/diff.js:81:25)
      at Object.diffElements (node_modules/unexpected-react/node_modules/unexpected-htmllike/lib/diff.js:71:12)
      at Object.diff (node_modules/unexpected-react/node_modules/unexpected-htmllike/lib/index.js:40:34)
      at Function.<anonymous> (node_modules/unexpected-react/lib/assertions.js:58:45)
      at executeExpect (node_modules/unexpected/lib/Unexpected.js:1070:50)
      at Unexpected.expect (node_modules/unexpected/lib/Unexpected.js:1078:22)
      at Context.<anonymous> (Select.spec.js:32:16)
      at Context.wrapper (node_modules/unexpected/lib/testFrameworkPatch.js:72:41)

```

**Comment by bruderstein** [comment]
I don't know why npm didn't install the `react-render-hook` correctly - I did publish a new version of it last night (about 11 hours ago from now), which corrected a small issue.  I've just cloned the `unexpected-react-example` repo, and done a `npm install`, `npm test` runs cleanly. What version of npm/node are you using?

As for the error, I can see under what circumstances this error could happen, but can't understand how it would be possible to arrive at those circumstances! For some reason, it can't find the children elements. Do you get this running `npm test`, or with some other way of running the tests?

**Comment by faceyspacey** [comment]
I'm running Node v4.2.2 on OSX. I did it several times.

git clone https://github.com/bruderstein/unexpected-react-example.git
cd unexpected-react-example
npm install
npm test

The SelectOption test works fine. ..This is really nice by the way. great work!

**Comment by faceyspacey** [comment]
here's what the error without `react-render-hook` manually added looks like:

```
Jamess-MacBook-Pro:unexpected-react-example jamesgillmore$ npm test

> unexpected-react-example@1.0.0 test /Users/jamesgillmore/Meteor/Apps/unexpected-react-example
> mocha --compilers js:babel-core/register src/tests/*.spec.js

module.js:339
    throw err;
    ^

Error: Cannot find module 'react-render-hook'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/jamesgillmore/Meteor/Apps/unexpected-react-example/node_modules/unexpected-react/lib/unexpected-react.js:3:24)
    at Module._compile (module.js:435:26)
    at Module._extensions..js (module.js:442:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/jamesgillmore/Meteor/Apps/unexpected-react-example/node_modules/babel-core/node_modules/babel-register/lib/node.js:135:7)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (Select.spec.js:4:25)
    at Module._compile (module.js:435:26)
    at loader (/Users/jamesgillmore/Meteor/Apps/unexpected-react-example/node_modules/babel-core/node_modules/babel-register/lib/node.js:127:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/jamesgillmore/Meteor/Apps/unexpected-react-example/node_modules/babel-core/node_modules/babel-register/lib/node.js:137:7)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at /Users/jamesgillmore/Meteor/Apps/unexpected-react-example/node_modules/mocha/lib/mocha.js:216:27
    at Array.forEach (native)
    at Mocha.loadFiles (/Users/jamesgillmore/Meteor/Apps/unexpected-react-example/node_modules/mocha/lib/mocha.js:213:14)
    at Mocha.run (/Users/jamesgillmore/Meteor/Apps/unexpected-react-example/node_modules/mocha/lib/mocha.js:453:10)
    at Object.<anonymous> (/Users/jamesgillmore/Meteor/Apps/unexpected-react-example/node_modules/mocha/bin/_mocha:393:18)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:467:10)
    at startup (node.js:136:18)
    at node.js:963:3
npm ERR! Test failed.  See above for more details.
```

**Comment by bruderstein** [comment]
Found it! react-render-hook was a dependency deeper down the tree, but it was missing directly from `unexpected-react`. It worked for me because I'm using npm3, which flattens the deps. Under npm2, you'd get that error.

Published 0.3.1 of `unexpected-react` which should fix this error.  (Feel free to reopen if it doesn't!)

Thanks for reporting.

**Comment by faceyspacey** [comment]
excellent, `Select` works now!

<img width="750" alt="screenshot 2015-11-26 23h 39m 02s" src="https://cloud.githubusercontent.com/assets/154732/11436163/e6181a84-9496-11e5-8187-fd2220a24c07.png">
